### DISCUSSION TITLE
How do you open the config screen?

### DISCUSSION THREAD

**Discussion by xp2-882030kgz010602** [P3pp3rF1y/SophisticatedBackpacks#984] [Q&A]
I want to know what the chance is that a mob spawns wearing a backpack, but I can't check the config because the button is grayed out.
![image](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/38965142/dbf3574b-a5d8-4853-99f6-21db83218541)

**Comment by KuschelKaktus88** [comment]
i think u just need to go into ur config files.

go in to ur modpack folder, click on configs, search for sophisticated backpacks and edit anything u want. dont forget to save what u have edited.
